README.md

```diff
@@ -86,51 +85,51 @@ list:
 -   Selection sort
 
 This illustrates something important about algorithms: you nearly always have a
-choice. There is no "one way" to solve a problem, no "right" way. Different
+choice. There is no one way to solve a problem. Different
 algorithms are better in different contexts, or with different constraints. It's
 up to you to consider the options and pick the one that best meets your needs.
 
 [Sorting Algorithms Visualized](https://www.toptal.com/developers/sorting-algorithms)
 
-### We do: Bubble Sort (5 min, 0:40)
+### We Do: Bubble Sort (5 minutes, 0:40)
 
-Next, we'll visualize bubble sorting with 7 volunteers. One person will play the role of recursive sorter.
+Next, we'll visualize bubble sorting with 7 volunteers. One person will play the role of the recursive sorter.
```

Comment:
so technically AP style is that if a number is less than 10 you spell it out....

(jk, trolling, but not really)